communication/src/allocator/mod.rs

```diff
@@ -32,6 +32,12 @@ pub trait AllocateBuilder : Send {
     fn build(self) -> Self::Allocator;
 }
 
+use std::any::Any;
+
+/// A type that can be sent along an allocated channel.
+pub trait Exchangeable : Send+Any+Bytesable+'static { }
+impl<T: Send+Any+Bytesable+'static> Exchangeable for T { }
```

Comment:
```suggestion
pub trait Exchangeable : Send+Any+Bytesable { }
impl<T: Send+Any+Bytesable> Exchangeable for T { }
```

`Any` implies `'static'`.